add more device info

devbobo opened the pull request:
Add Model, SerialNumber, FirmwareRevision, HardwareRevision to device info.

rudders commented:
Thanks mate - I'll update the latest index.js and compare and then process your pull - thanks.!

devbobo commented:
I've just added another commit, with the following changes...
* added support for WeMo Motion, including a "no motion" timer which delays sending an "off" signal for X number of seconds. This allows for automation using ON and OFF triggers.
* add support for broadcasting/notifying changes. This is predominantly for the WeMo motion, but also works well for updating local control of a WeMo switch (eg. toggling the button on the unit)
* Removed IF statements using `enddevice` in favour of `device.deviceType` for better readability
* Changed logic to prevent the additional of unused variables and event triggers.

@rudders, I don't have any WeMo lights (or light switches), so I'm hoping my changes haven't broken anything...if you can test that would be greatly appreciated. :+1:

rudders commented:
unfortunately enddevice doesn't contain this data. Here's a sample.
```javascript
{ friendlyName: 'Back Room',
HomeBridge-0      deviceId: '94103EF6BF43E6E5',
HomeBridge-0      currentState: [ '1', '22:0', '', '', '' ],
HomeBridge-0      capabilities: [ '10006', '10008', '30008', '30009', '3000A' ],
HomeBridge-0      internalState: 
HomeBridge-0       { '10006': '1',
HomeBridge-0         '10008': '22:0',
HomeBridge-0         '30008': '',
HomeBridge-0         '30009': '',
HomeBridge-0         '3000A': '' },
HomeBridge-0      deviceType: 'dimmableLight' }
```
